### ISSUE TITLE
Sorting by default the output?

### ISSUE THREAD

**Issue by kucharskim** [semarie/sysclean#14]
What do you think about sorting the output by default. My only way of executing `sysclean` is always as `sysclean | sort` so I can easily see what to ignore and what to remove, based on whole subpaths. Do you think it would be worth to add sorting by default or add `-s` or similar for sorted output?

**Comment by semarie** [comment]
the output isn't sorted as it would implies to bufferize the output (wait until the end to see something).

currently, sysclean uses `File::Find::find()` to do the traversal. from perldoc, it is a depth-first search, so the output should be somehow already sorted by subpaths ?

**Comment by kucharskim** [comment]
Yes, it's true that is it's somehow sorted. I don't user `-i` so my clean up list is usually long and most of list generated by `sysclean` is actually NOT to be cleaned up and those files are important to me. So, in practice I sort the `sysclean` output to clearly see what is to be remove and what it not.
